pkg/experiment/ingester/client/client.go

```
@@ -186,18 +186,17 @@ func (c *Client) Service() services.Service { return c.service }
 func (c *Client) starting(ctx context.Context) error {
 	// Warm up connections. The pool does not do this.
 	instances, err := c.ring.GetAllHealthy(ring.Reporting)
-	if err != nil {
-		return fmt.Errorf("get all healthy instances: %w", err)
-	}
-	var wg sync.WaitGroup
-	for _, x := range instances.Instances {
-		wg.Add(1)
-		go func(x ring.InstanceDesc) {
-			defer wg.Done()
-			_, _ = c.pool.GetClientFor(x.Addr)
-		}(x)
+	if err == nil {
```

consider adding a comment why we ignore error in the else case